spec/controllers/api/schedules/schedules_create_spec.rb

```diff
@@ -18,5 +19,16 @@
       expect(response.status).to eq(200)
       expect(before < Schedule.count).to be_truthy
     end
+
+    it 'handles missing schedule_id' do
+      sign_in user
+      input = { start_time: '2015-02-17T15:16:17.000Z',
+                end_time: '2099-02-17T18:19:20.000Z',
+                repeat: 4,
+                time_unit: 'minutely' }
+      post :create, input
+      expect(response.status).to eq(400)
+      expect(json[:error]).to include("forgot to provide an `*_id` attribute")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.